Vagrantfile

```diff
@@ -106,6 +106,7 @@ Vagrant.configure("2") do |config|
       # Only execute once the Ansible provisioner,
       # when all the machines are up and ready.
       if i == $num_instances
+        config.vm.provision :shell, inline: '[[ -x "$(command -v yum)" ]] && ifup eth1', privileged: true
```

Comment:
Looks like this might get fixed in Vagrant 1.9.2... 

https://github.com/mitchellh/vagrant/pull/8148